File: TFC-AppEventos/TFC.AppEventos.Application.DTO/TournamentDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TFC.AppEventos.Application.DTO
{
    public class TournamentDto
    {
        public int TournamentId { get; set; }
        public string location { get; set; }
        public string Arena { get; set; }
        public string StartDate { get; set; }
        public string EndDate { get; set; }
        public string SportType { get; set; }
        public int OrganizerId { get; set; }
    }
}

File: Flutter/app_eventos/lib/core/models/dto/tournament_dto.dart
class TournamentDto {
  final int tournamentId;
  final String location;
  final String arena;
  final String startDate;
  final String endDate;
  final String sportType;
  final int organizerId;

  TournamentDto({
    required this.tournamentId,
    required this.location,
    required this.arena,
    required this.startDate,
    required this.endDate,
    required this.sportType,
    required this.organizerId,
  });

  factory TournamentDto.fromJson(Map<String, dynamic> json) => TournamentDto(
    tournamentId: json['tournamentId'],
    location: json['location'],
    startDate: json['startDate'],
    arena: json['arena'],
    endDate: json['endDate'],
    sportType: json['sportType'],
    organizerId: json['organizerId'],
  );
}